CHANGELOG.md

```diff
@@ -11,6 +14,8 @@
     - Significant speed increase available via the C++ backend
     - Returned geometry is correctly offset where binning/cropping moves the origin
   - Github Actions:
```

Comment:
It seems odd having a GHA section in the CIL release notes. It doesn't have any impact on CIL, and it doesn't go live at release. It is nice to keep a log of changes, but I'm not sure here is the right place for it.